test/src/xquery/xquery3/fn.xql

```diff
@@ -34,23 +34,28 @@ function fnt:has-children-contextItem-noChildren() {
     <a/>/has-children()
 };
 
+(: ERROR: It is a dynamic error if evaluation of an expression relies on some part of
+          the dynamic context that has not been assigned a value. Context item is absent
 declare
     %test:assertFalse
 function fnt:has-children-contextItem-empty() {
     ()/has-children()
 };
+:)
 
 declare
     %test:assertError("XPDY0002")
 function fnt:has-children-contextItem-absent() {
     has-children()
 };
 
+(: fail with XPTY0019
```

Comment:
Ha! Here's an example - https://github.com/eXist-db/exist/pull/1177 - a PR awaiting review in fact.